lib/services/sf.js

```diff
@@ -1457,7 +1457,7 @@ StateConnected.prototype.destroy = function (options)
       callback: function (err, body)
       {
         // if the destroy request succeeded, we're disconnected
```

Comment:
should we change the comment? `... or the session already expired ...`

Reply:
done